src/main/java/cz/cvut/kbss/study/rest/PatientRecordController.java

```diff
@@ -42,10 +35,12 @@ public class PatientRecordController extends BaseController {
     private final PatientRecordService recordService;
 
     private final ApplicationEventPublisher eventPublisher;
+    private final HttpServletResponse httpServletResponse;
 
-    public PatientRecordController(PatientRecordService recordService, ApplicationEventPublisher eventPublisher) {
+    public PatientRecordController(PatientRecordService recordService, ApplicationEventPublisher eventPublisher, HttpServletResponse httpServletResponse) {
         this.recordService = recordService;
         this.eventPublisher = eventPublisher;
+        this.httpServletResponse = httpServletResponse;
```

Comment:
Why do we need this, if we inject it into actual call of `exportRecords()`

Reply:
@blcham
Removed